docs/Concepts/Privacy/PrivateTransactionManager.md

```diff
@@ -6,14 +6,47 @@ private transactions.
 The Private Transaction Manager is a separate component that is concerned with the storing and distribution
 of encrypted private transaction data between recipients of a private transaction.
 
+The Private Transaction Manager has two distinct components:
+
+* Transaction Manager
+* Enclave
+
 To enable private transactions, use the `PRIVATE_CONFIG` environment variable when starting a GoQuorum
-node to provide the node with the path to the Privacy Manager's `.ipc` socket, e.g.:
+node to provide the path to the Privacy Manager's `.ipc` socket.
+
+Use the `PRIVATE_CONFIG` environment variable either with a direct path to the IPC socket or with a path to a configuration file as explained in the following sections. 
```

Comment:
@NicolasMassart Your amendment is now duplicating the text "Use the `PRIVATE_CONFIG` environment variable'".

Reply:
We can repeat this kind of things if it makes the text easier to read. Sometimes using references like "this" or "previous" is not clear enough for non english speakers of when translated using Google Translate (lof of Chinese users using Translate for instance).